ocs_samples/basic_samples/SDS/DotNet/SdsClientLibraries/SdsClientLibraries/readme.md

```diff
@@ -1,19 +1,12 @@
 .NET Samples 
```

Comment:
Not specific to these changes, but I am seeing the markdown on the code blocks when using:
.. code:: cs, shouldn't we be using the triple backtick https://help.github.com/en/articles/creating-and-highlighting-code-blocks

Reply:
Yep. I was not sure if @derekendres had that in there for a specific reason.

Reply:
Yeah.  Will add that to changes to do :).  Lots of little things...